apps/fares/lib/repo.ex

```diff
@@ -38,4 +38,24 @@ defmodule Fares.Repo do
       (!media || media in fare.media) && match?(^opts, Map.take(fare, Map.keys(opts)))
     end)
   end
+
+  @spec for_fare_class(Routes.Route.gtfs_fare_class(), Keyword.t()) :: [Fare.t()]
+  def for_fare_class(fare_class, opts \\ []) do
+    opts = Keyword.merge(opts, fare_class_opts(fare_class))
+    all(opts)
+  end
+
+  def fare_class_opts(:free_fare), do: [name: :free_fare]
+
+  def fare_class_opts(:express_bus_fare), do: [name: :express_bus]
+
+  def fare_class_opts(:local_bus_fare), do: [name: :local_bus]
+
+  def fare_class_opts(:rapid_transit_fare), do: [mode: :subway]
+
+  def fare_class_opts(:ferry_fare), do: [mode: :ferry]
+
+  def fare_class_opts(:commuter_rail_fare), do: [mode: :commuter_rail]
+
+  def fare_class_opts(_), do: []
```

Comment:
question(non-blocking): This seems like a hack to map the existing mode fares map (`fare_info`) to the new `fare_class` property.  Is that right?  Is there a plan to move this towards reworking the `fare_info` map to use `fare_class` instead of mode/name/zone filters?

Reply:
Yeah, you've got it exactly right. We can definitely use `fare_class` to improve our `fare_info` maps, but I figured that could wait, especially if we're gonna eventually incorporate fares info from the API down the line (whenever that happens).